cal/tests/unit/v1/network/drivers/test_openstack_driver.py

```diff
@@ -0,0 +1,355 @@
+""" OpenstackDriver for Network
+    based on NetworkDriver
+"""
+
+
+import mock
+from cal.tests import base
+from keystoneauth1.exceptions.base import ClientException
+from cal.v1.network.drivers.openstack import OpenstackDriver
```

Comment:
the import order is not good, all outside import statement need to separate from inside ones.